service/src/main/kotlin/fi/espoo/evaka/shared/async/AsyncJobQueries.kt

```diff
@@ -44,14 +44,14 @@ AND type = ANY(:jobTypes)
         .mapTo<Int>()
         .one()
 
-fun <T : AsyncJobPayload> Database.Transaction.claimJob(jobTypes: Collection<AsyncJobType<out T>>): ClaimedJobRef<out T>? =
+fun <T : AsyncJobPayload> Database.Transaction.claimJob(now: HelsinkiDateTime, jobTypes: Collection<AsyncJobType<out T>>): ClaimedJobRef<out T>? =
     createUpdate(
         // language=SQL
         """
 WITH claimed_job AS (
   SELECT id
   FROM async_job
-  WHERE run_at < now()
+  WHERE run_at <= :now
```

Comment:
Is this functional change intended?

Reply:
Yes, that's why it's in a separate commit: https://github.com/espoon-voltti/evaka/pull/2857/commits/a179e77d211ca0d3199636652b0ae39c6d548841